doc/index.md

```diff
@@ -89,6 +89,7 @@ interface ISdkConfig {
     disableAutoAnswer?: boolean;
     logLevel?: LogLevels;
     logger?: ILogger;
```

Comment:
I'm worried this may be considered a breaking change because currently, we are allowing users to pass in an `ILogger` which accepts a boolean as the last parameter. By taking that away and requiring `ILogMessageOptions` as the third param, it could cause consumers' typescript to fail to compile or worse start throwing errors when the logger tries to access `opts.skipServer` on a `boolean` (but even if it handles the error, it won't respect the `true` flag to skip server).

Reply:
you're right, I didn't even think about it. We'll include this in 7.x